tests/unit/ast/nodes/test_fold_binop_decimal.py

```diff
@@ -81,11 +84,13 @@ def foo({input_value}) -> decimal:
     try:
         vyper_ast = parse_and_fold(literal_op)
         expr = vyper_ast.body[0].value
+
+        # check invalid intermediate values
         ExprVisitor().visit(expr, DecimalT())
+
         new_node = expr.get_folded_value()
         expected = new_node.value
-        lo, hi = DecimalT().decimal_bounds
-        is_valid = lo <= expected < hi
```

Comment:
This no longer needs to be checked?

Reply:
it's checked by `ExprVisitor().visit()`, which will throw an exception if oob